server/leader.go

```diff
@@ -406,3 +407,25 @@ func (s *Server) reloadConfigFromKV() error {
 	}
 	return nil
 }
+
+//checkConfigChanged will check whether the config of leader
+//changed, it is used for follower.
+func (s *Server) checkConfigChanged() {
+	for {
+		//follower watch config change and apply it
+		configHash, err := s.kv.LoadConfigHash()
+		if err != nil {
+			log.Error("load config hash failed", zap.Error(err))
+			break
+		}
+
+		if strings.Compare(*s.scheduleOpt.loadConfigHash(), configHash) != 0 {
+			err := s.reloadConfigFromKV()
+			if err != nil {
+				log.Error("load config failed", zap.Error(err))
+				break
+			}
+		}
+		time.Sleep(200 * time.Millisecond)
```

Comment:
I think it is too frequent to read from `KV`, maybe we can use `Watch` API like the leader key to listen key changed.

Reply:
:ok_hand:

Reply:
Hi @bradyjoestar, after we discussed, there is a better solution. Because we not only PD need to dynamically load the configuration, but also TiKV needs. we can add a service in `gRPC` that provides the `Watch` function. If you are interested, you can send your `wechat` to my email. chenshuning@pingcap.com

Reply:
@nolunch Have send to your email. It sounds cool :smile_cat: